tests/urls.php

```diff
@@ -17,58 +17,62 @@ public function setUp(): void {
 		$this->event_factory    = new Event_Factory();
 		$this->event_repository = new Event_Repository( new Attendee_Repository() );
 		$this->set_normal_user_as_current();
+
+		if ( ! defined( 'GP_URL_BASE' ) ) {
+			define( 'GP_URL_BASE', '/' );
+		}
```

Comment:
Oh, I wanted to suggest if we could test both, one with a GP_URL_BASE defined, and one without. So that it shows that it properly generates URLs for both cases.

Reply:
AFAIK you can just define it on a per test basis.

Reply:
Gotcha, fixed.

Reply:
> AFAIK you can just define it on a per test basis, even though it's a constant; the next test will run with the constant undefined again.

That's what I attempted to do indeed, but it's not the case, sadly. Because it's a PHP constant, once it's defined it can't be modified or unset. I also tried adding it to a different file, but that didn't work either.

I think since all tests are run in a single php process, once that constant is defined, it will keep the value until the process ends.